crates/revm/CHANGELOG.md

```diff
@@ -261,7 +261,7 @@ Github Changelog:
 * de83db6 - fix: feature flags (#330) (2 weeks ago) <Wodann>
 * b60269c - `revm`: mark `with-serde` feature as deprecated (#328) (2 weeks ago) <Enrique Ortiz>
 * 63bf475 - make load_account pub (#325) (3 weeks ago) <rakita>
-* 0ef0197 - Cleanup, move hot fields toggether in Interpreter (#321) (3 weeks ago) <rakita>
+* 0ef0197 - Clean-up, move hot fields together in Interpreter (#321) (3 weeks ago) <rakita>
```

Comment:
This is git commit, should not be changed

Reply:
yes you are right I will update my pr